phpcs.xml

```diff
@@ -27,6 +27,14 @@
 		</properties>
 	</rule>
 
+	<rule ref="WordPress.Security.EscapeOutput">
+		<properties>
+			<property name="customEscapingFunctions" type="array">
+				<element value="esc_xml"/>
```

Comment:
Should this not be added upstream?
https://developer.wordpress.org/reference/functions/esc_xml/

Reply:
You talk to Juliette.
https://github.com/WordPress/WordPress-Coding-Standards/issues/1937

Reply:
WPCS 3.0 appears to have implemented this now. 

I'll implement this workaround and remove it once I upgrade wpcs-tsf for TSF v4.3.1+.

Reply:
All right.